docs/quickstarts/nextjs.mdx

```diff
@@ -118,68 +120,113 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
   </Tab>
 </Tabs>
 
-### Require authentication to access your app
+### Add Middleware to your app
 
-Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access.
+Middleware will allow you to access user auth state on any route or page in your application. `clerkMiddleware()` also provides you with the ability to protect routes, but by default it does not protect any routes.
 
-Create a `middleware.ts` file. If your application uses the `src/` directory, your `middleware.ts` file should be placed inside the `src/` folder. If you are not using a `src/` directory, place the `middleware.ts` file in your root directory alongside `.env.local`.
+Create a middleware.ts file. If your application uses the src/ directory, your middleware.ts file should be placed inside the src/ folder. If you are not using a src/ directory, place the middleware.ts file in your root directory alongside .env.local.
 
-Copy and paste the following code into your `middleware.ts`. With this, your entire application is protected.
+Copy and paste the following code into your middleware.ts. 
 
 ```tsx filename="middleware.ts"
-import { authMiddleware } from "@clerk/nextjs";
+import { clerkMiddleware } from "@clerk/nextjs/server";
 
-// This example protects all routes including api/trpc routes
-// Please edit this to allow other routes to be public as needed.
-// See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
-export default authMiddleware({});
+export default clerkMiddleware()
 
 export const config = {
-  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
+  matcher: ['/((?!.+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'],
 };
 ```
 
-**Try accessing your app now by visiting [`http://localhost:3000`](http://localhost:3000).** The Middleware will redirect you to your Sign Up page, provided by Clerk's [Account Portal](/docs/account-portal/overview) feature. If you want to make other routes public, check out the [`authMiddleware` reference page](/docs/references/nextjs/auth-middleware). Go ahead and sign up to get access to your application. Your app is now fully protected by Clerk!
+### Use Clerk components to protect your content
 
-### Embed the `<UserButton />`
+Clerk offers a set of [prebuilt components](/docs/components/overview) that can gate access to content based on whether a user is signed in and has the correct permissions. You can use Clerk's [`<SignedIn>`](/docs/components/control/signed-in) and [`<SignedOut>`](/docs/components/control/signed-out) components to restrict access to content and even whole pages. The `<SignedIn>` component requires the user be signed in to view the content inside the tags. The `<SignedOut>` components requires the user to be signed out.

```

Comment:
Thoughts on restructuring? 
```suggestion
Clerk provides prebuilt components that control content access based on user authentication and permissions. These components can be used to restrict access to specific content or entire pages.

To manage content visibility for signed-in users, use the `<SignedIn>` component:
- Enclose content within the `<SignedIn>` tags to make it viewable only by users who are currently signed in.
- Content outside of these tags remains accessible to both signed-in and signed-out users.

To manage content visibility for signed-out users, use the `<SignedOut>` component:
- Enclose content within the `<SignedOut>` tags to make it visible only to users who are currently not signed in.
- Content outside of these tags is only accessible to signed-in users.

For more information on these components, refer to the following documentation:
- Overview of Clerk's Prebuilt Components: /docs/components/overview
- Control Access with `<SignedIn>`: /docs/components/control/signed-in
- Control Access with `<SignedOut>`: /docs/components/control/signed-out

```

Reply:
Could use **bold** in a few places but wasnt as sure about that.

Reply:
I like it, but in my opinion, we can omit the last section "For more information on these components" and simply link their first instances. So, where you have "To manage content visibility for signed-in users, use the `<SignedIn>` component" we can put the reference link there.